### PR TITLE
Correcting question-044.md

### PR DESCRIPTION
**Pull request by dostonhamrakulov**
## PR Type

- [ ] Correcting the question 44:
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

## Screenshots

Screenshot 2024-06-09 at 21 11 37

### CODE REVIEW

File: content/questions/actions/question-044.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ draft: false
 ---
 
 
-> https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#about-workflow-artifacts
-1. [x] You cannot access `artifacts` that were created in a different workflow run
-1. [ ] Use the `actions/download-artifact` action.
+> https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#downloading-or-deleting-artifacts
+1. [ ] You cannot access `artifacts` that were created in a different workflow run
+1. [x] Use the `actions/download-artifact` action.
```
Comment (FidelusAleksander) [inline content/questions/actions/question-044.md:11]: As stated in the screenshot you pointed out yourself - `actions/download-artifact` works only if the artifact was created in the same workflow run. The question asks about accessing artifacts from **previously triggered workflow run**.

As for me there is nothing to fix here. Please explain if I'm missing something

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Hi @dostonhamrakulov 👋 

Thanks for taking interest in the project- I've left a comment regarding the proposed changes

File: content/questions/actions/question-044.md (hunk 1 above)
Comment (dostonhamrakulov) [reply content/questions/actions/question-044.md:11]: Hi,
hm I think I misunderstood question or the question is so tricky to grab its meaning. I would put like: 
To access an `artifact` that was created in another workflow run you can:

But it is fine, thank you for clarifying

**Comment by dostonhamrakulov** [comment]
To access an `artifact` that was created in another, previously triggered workflow run you can:

**Pull request closed without merging**